README.md

```diff
@@ -1,13 +1,28 @@
-# Grafana Data Source Plugin Template
+# Chaos Mesh Data Source
 
-[![CircleCI](https://circleci.com/gh/grafana/simple-datasource/tree/master.svg?style=svg)](https://circleci.com/gh/grafana/simple-datasource/tree/master)
+Visualize Chaos Mesh Events with Grafana
 
-This template is a starting point for building Grafana Data Source Plugins
+![screenshot](assets/example.png)
 
-## What is Grafana Data Source Plugin?
-Grafana supports a wide range of data sources, including Prometheus, MySQL, and even Datadog. There’s a good chance you can already visualize metrics from the systems you have set up. In some cases, though, you already have an in-house metrics solution that you’d like to add to your Grafana dashboards. Grafana Data Source Plugins enables integrating such solutions with Grafana.
+## Getting Started
+
+### Use Grafana image with Chaos Mesh data source
+
+Since Chaos Mesh data source is not officially released, it is not supported to install it from [Grafana Plugins Hub](https://grafana.com/grafana/plugins).
+
+But you can still try to use the customized Grafana Docker image, with the Chaos Mesh data source built-in.
+
+```bash
+docker pull yeya24/grafana:chaos
```

Comment:
We can later push to pingcap/grafana:chaos, since chaosmesh is already token..

Reply:
This image is only for testing. After this data source is officially ready, we can directly install it at Grafana. So no additional images are required.